mail_notification_custom_subject/migrations/16.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,20 @@
+# pylint: disable=C7902
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    env.cr.execute(
+        """
+        UPDATE mail_message_custom_subject
+        SET subject_template=REPLACE(
+                REPLACE(
+                subject_template,
+                '${',
+                '{{'),
+                '}',
+                '}}'
+            )
```

Comment:
This migration script is used to migrate 14 > 15, I think we could remove this. (remove migrations folder)

> Remove any possible migration script from previous version (in a nutshell, remove migrations folder inside the module if exists). https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0